diff --git a/pkgs/equicord.nix b/pkgs/equicord.nix
index 4cf1b60..b1416ca 100644
--- a/pkgs/equicord.nix
+++ b/pkgs/equicord.nix
@@ -17,7 +17,7 @@ let
   version = "1.15.4.0-2026-09-03";
   rev = "b90a13be8e0636837e9bfdc8f18cc40cf8190962";
   hash = "sha256-Bu1226PDuCmY8w7RKTMZJzsLQiF0URuABZGxgo78H7k=";
-  pnpmDepsHash = "sha256-VW4VKmM/12wl0sMFB9hG9K+sXjYVxlafu14IplGL/sw=";
+  pnpmDepsHash = "sha256-hBZHHB5kRkNqep5vWMMnwIblNCAOZvLotDjJUJd9iMU=";
   inherit (equicord.src) owner repo;
   src = fetchFromGitHub {
     inherit
